python/lsst/meas/algorithms/utils.py

```diff
@@ -224,7 +224,7 @@ def showPsfCandidates(exposure, psfCellSet, psf=None, frame=None, normalize=True
                 # residuals using spatial model
                 try:
                     subtractPsf(psf, im, xc, yc)
-                except:
+                except Exception:
```

Comment:
Please check if there's an obvious exception(s) raised by `subtractPSf` here.

Reply:
Here is the full documentation for subtractPsf:
```
/**
 * Subtract a PSF from an image at a given position
 */
template<typename MaskedImageT>
double subtractPsf(afwDetection::Psf const& psf,      ///< the PSF to subtract
                   MaskedImageT *data,                ///< Image to subtract PSF from
                   double x,                          ///< column position
                   double y,                          ///< row position
                   double psfFlux                     ///< object's PSF flux (if not NaN)
                  )
```
I think this is a clear case where `except Exception:` is actually reasonable. This means "make a best effort to subtract the PSF and continue if it fails. Fortunately this is plotting code.